metrics/opentelemetry.go

```diff
@@ -32,8 +32,33 @@ var (
 	RetrievalErrorNoOnlineCount              = stats.Int64("retrieval_error_noonline_total", "The number of retrieval errors for 'miner is not accepting online retrieval deals'", stats.UnitDimensionless)
 	RetrievalErrorUnconfirmedCount           = stats.Int64("retrieval_error_unconfirmed_total", "The number of retrieval errors for 'unconfirmed block transfer'", stats.UnitDimensionless)
 	RetrievalErrorOtherCount                 = stats.Int64("retrieval_error_other_total", "The number of retrieval errors with uncategorized causes", stats.UnitDimensionless)
+	QueryErrorFailedToDialCount              = stats.Int64("query_error_failed_to_dial", "The number of query errors because we coult not connect to the provider", stats.UnitDimensionless)
+	QueryErrorFailedToOpenStreamCount        = stats.Int64("query_error_failed_to_open_stream", "The number of query errors where the miner did not respond on the retrieval protocol", stats.UnitDimensionless)
+	QueryErrorResponseEOFCount               = stats.Int64("query_error_response_eof", "The number of query errors because where the response terminated early", stats.UnitDimensionless)
+	QueryErrorResponseStreamResetCount       = stats.Int64("query_error_response_eof", "The number of query errors because where the response stream was reset", stats.UnitDimensionless)
```

Comment:
```suggestion
	QueryErrorResponseStreamResetCount       = stats.Int64("query_error_response_stream_reset", "The number of query errors because where the response stream was reset", stats.UnitDimensionless)
```